x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -240,6 +242,23 @@ export const EndpointList = () => {
     }
   );
 
+  const backToEndpointList: PolicyDetailsRouteState['backLink'] = useMemo(() => {
```

Comment:
From our Grooming meeting: if it is decided to now release the Policy List feature, then we'll need to revert this back... or, you can add a feature flag check here and adjust as needed.

/cc @kevinlog